exchange/exchange.go

```diff
@@ -119,9 +124,25 @@ func (e *exchange) HoldAuction(ctx context.Context, bidRequest *openrtb.BidReque
 		e.me.RecordImps(impLabels)
 	}
 
+	// Make our best guess if GDPR applies
+	usersyncIfAmbiguous := e.UsersyncIfAmbiguous
+	var geo *openrtb.Geo = nil
+	if bidRequest.User != nil && bidRequest.User.Geo != nil {
+		geo = bidRequest.User.Geo
+	} else if bidRequest.Device != nil && bidRequest.Device.Geo != nil {
+		geo = bidRequest.Device.Geo
+	}
+	if geo != nil {
+		if _, found := e.eeaCountries[geo.Country]; found {
+			usersyncIfAmbiguous = false
```

Comment:
Please add a comment indicating that gdpr is always enforced if a country is in this list.

Reply:
done